src/v/cluster/self_test/diskcheck.h

```diff
@@ -96,27 +96,25 @@ class diskcheck final {
 
     ss::future<std::vector<self_test_result>> initialize_benchmark(ss::sstring);
     ss::future<std::vector<self_test_result>>
-    run_configured_benchmarks(ss::file&);
+    run_configured_benchmarks(std::vector<ss::file>&);
```

Comment:
should be `const std::vector<ss:file> &` I think (as well as other functions which take this).

Reply:
Has to be non-const actually as later we need a non-const file to call dma_write etc on it.

Reply:
wait lol is deep

Reply:
I had thought it was deep, did a very brief search to check it but was led astray